sqlalchemy_ibmi/reflection.py

```diff
@@ -60,10 +60,7 @@ def denormalize_name(self, name):
             else:
                 name = codecs.decode(name)
         else:
-            if version_info[0] < 3:
-                name = unicode(name)
-            else:
-                name = str(name)
+            name = str(name)
```

Comment:
This doesn't actually work as it was coded, since a `str` in Python 3 is actually Unicode. Of course, I don't think we have the restriction of not supporting Unicode binds, so the whole if condition is unneeded.